storage-model/src/main/java/no/unit/nva/publication/storage/model/DoiRequest.java

```diff
@@ -43,18 +50,18 @@
     private SortableIdentifier identifier;
 
     @JsonCreator
-    public DoiRequest(@JsonProperty("identifier") SortableIdentifier identifier,
-                      @JsonProperty("resourceIdentifier") SortableIdentifier resourceIdentifier,
-                      @JsonProperty("resourceTitle") String resourceTitle,
-                      @JsonProperty("owner") String owner,
-                      @JsonProperty("customerId") URI customerId,
-                      @JsonProperty("status") DoiRequestStatus status,
-                      @JsonProperty("resourceStatus") PublicationStatus resourceStatus,
-                      @JsonProperty("modifiedDate") Instant modifiedDate,
-                      @JsonProperty("createdDate") Instant createdDate) {
+    private DoiRequest(@JsonProperty("identifier") SortableIdentifier identifier,
```

Comment:
Make Json creator private and use static methods for creating objects in the code